CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 1.9.0
+
+## Features and improvements
+* Map API functions such as `easeTo` and `flyTo` now support `padding: PaddingOptions` which lets developers shift the center of perspective for a map when building floating sidebars ([#8638](https://github.com/mapbox/mapbox-gl-js/pull/8638))
+* Introduce `within` expression ([#9352](https://github.com/mapbox/mapbox-gl-js/pull/9352))
+
+### 🍏 Improvements
+* Add changelog policy to contribution guidelines ([#9378](https://github.com/mapbox/mapbox-gl-js/pull/9378))
+* Reduce bundle size and improve visual quality of `showTileBoundaries` debug text ([#9267](https://github.com/mapbox/mapbox-gl-js/pull/9267))
+* Clarify differences in `symbol-sort-key` with and without overlap ([#9379](https://github.com/mapbox/mapbox-gl-js/pull/9379))
+* Add browser testing with Selenium/WebDriver ([#9245](https://github.com/mapbox/mapbox-gl-js/pull/9245))
+* Slightly improve GPU memory footprint ([#9377](https://github.com/mapbox/mapbox-gl-js/pull/9377))
+* Results from `queryRenderedFeatures` now have evaluated property values rather than raw expressions ([#9198](https://github.com/mapbox/mapbox-gl-js/pull/9198))
+* `LngLatBounds.extend` now accepts `LngLatLike` and `LngLatBoundsLike` as well. ([#9293](https://github.com/mapbox/mapbox-gl-js/pull/9293))
+* Clarify GeolocateControl interaction states ([#9324](https://github.com/mapbox/mapbox-gl-js/pull/9324))
```

Comment:
This looks like a documentation change, do we want to state it in the changelog?